__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const request = require("supertest");
+const db = require("../db/connection");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+
+afterAll(() => db.end());
+beforeEach(() => seed(data));
+
+describe("/api", () => {
+  test("400: sends an appropriate error message if given an invalid path", () => {
+    return request(app)
+      .get("/api/toopics")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid Path");
+      });
+  });
+  describe("/topics", () => {
+    describe("GET", () => {
+      test("200: sends an array of topic objects with the correct properties", () => {
+        return request(app)
+          .get("/api/topics")
+          .expect(200)
+          .then(({ body }) => {
+            expect(Array.isArray(body.topics)).toBe(true);
+            body.topics.forEach((topic) => {
+              expect("slug" in topic).toBe(true);
+              expect("description" in topic).toBe(true);
```

Comment:
This is good, it would also be beneficial to assert that the array is not empty, as currently an empty array `[]` would pass the test block because it would skip past the forEach